### ISSUE TITLE
Cache downloaded files

### ISSUE THREAD

**Issue by raulfraile** [symfony/symfony-installer#23]
If a user tries to install the same version of Symfony several times, the file should be downloaded only once and saved in, for example, `~/.symfony/cache`.

In case the installer provides a cache system, it should be added a new command to clean the cache.

**Comment by Pierstoval** [comment]
Is this issue solved?

**Comment by javiereguiluz** [comment]
The downloaded file is not cached, but the temp files are removed "most of the times" (with some fatal errors we cannot remove them).

**Comment by javiereguiluz** [comment]
Closing as "won't fix" because now that [Symfony Flex](https://github.com/symfony/flex) replaces the Symfony Installer, we won't add new features to the installer. Thanks for understanding it.
